lineapy/transformer/node_transformer.py

```diff
@@ -739,3 +764,49 @@ def get_source(self, node: ast.AST) -> Optional[SourceLocation]:
             end_lineno=node.end_lineno,  # type: ignore
             end_col_offset=node.end_col_offset,  # type: ignore
         )
+
+    def get_else_source(self, node: ast.If) -> Optional[LiteralNode]:
+        body_source = self.get_source(node.body[-1])
+        orelse_source = (
+            self.get_source(node.orelse[0]) if node.orelse else None
+        )
+        assert body_source, "Body of If/Else must have at least one statement"
```

Comment:
dont think this is needed. add a test case that hits this line if you can find one.